### DISCUSSION TITLE
File opener protocol

### DISCUSSION THREAD

**Discussion by farskid** [zed-industries/zed#6516] [Feature Requests]
With VS Code, you can use `vscode://file//PATH_TO_FILE` in the browser to open a file in VS Code. Similarly, WebStorm has the `webstorm://open?file=PATH_TO_FILE` way of doing it. Is there such protocol with Zed too?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
